### PR TITLE
Readme testing update

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Discussed with @jamesdonoh.

Adding more explanation about the environment variables and smoke tests to the main README.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,21 +280,41 @@ We have [Jest](https://facebook.github.io/jest) unit tests that can be run with
...
+| CYPRESS_ONLY_SERVICE | Restricts to running only the specified service                                                               | A single service i.e. `CYPRESS_ONLY_SERVICE=urdu` |
+| CYPRESS_APP_ENV      | Runs the tests in a specific environment                                                                      | `test`, `local`, `live`                           |
+| CYPRESS_SMOKE        | Runs only smoke tests if true                                                                                 | boolean                                           |
+| CYPRESS_UK           | See [running e2es in the UK against Live](https://github.com/bbc/simorgh/#running-e2e-in-the-uk-against-live) | boolean                                           |
+| CYPRESS_SKIP_EU      | See [running e2es outside EU](https://github.com/bbc/simorgh/#running-e2e-outside-eu)                         | boolean                                           |
+
+These commands can be run in combination.
+
+#### Smoke tests
```
Comment (amywalkerdev) [inline README.md]: I would rename this full suite of tests or something similar, as we are already stating above that `npm run test:e2e` only runs the smoke tests.

**Review by amywalkerdev — CHANGES_REQUESTED** [review]

**Review by amywalkerdev — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -280,21 +280,41 @@ We have [Jest](https://facebook.github.io/jest) unit tests that can be run with
...
 
+#### Environment variables
+
+There are several environment variables you can use with our test suite, which are:
+
+| Envionment variable  | Effect                                                                                                        | Possible values                                   |
+| -------------------- | ------------------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
+| CYPRESS_ONLY_SERVICE | Restricts to running only the specified service                                                               | A single service i.e. `CYPRESS_ONLY_SERVICE=urdu` |
+| CYPRESS_APP_ENV      | Runs the tests in a specific environment                                                                      | `test`, `local`, `live`                           |
+| CYPRESS_SMOKE        | Runs only smoke tests if true                                                                                 | boolean                                           |
```
Comment (EinsteinNjoroge) [inline README.md]:
```suggestion
| CYPRESS_SMOKE        | Runs only smoke tests if true                                                                                 | `true`, `false`                                          |
```

File: README.md [hunk 3]
```diff
@@ -280,21 +280,41 @@ We have [Jest](https://facebook.github.io/jest) unit tests that can be run with
...
+| CYPRESS_UK           | See [running e2es in the UK against Live](https://github.com/bbc/simorgh/#running-e2e-in-the-uk-against-live) | boolean                                           |
```
Comment (EinsteinNjoroge) [inline README.md]:
```suggestion
| CYPRESS_UK           | See [running e2es in the UK against Live](https://github.com/bbc/simorgh/#running-e2e-in-the-uk-against-live) | `true`, `false`                                           |
```

File: README.md [hunk 4]
```diff
@@ -280,21 +280,41 @@ We have [Jest](https://facebook.github.io/jest) unit tests that can be run with
...
+| CYPRESS_SKIP_EU      | See [running e2es outside EU](https://github.com/bbc/simorgh/#running-e2e-outside-eu)                         | boolean                                           |
```
Comment (EinsteinNjoroge) [inline README.md]:
```suggestion
| CYPRESS_SKIP_EU      | See [running e2es outside EU](https://github.com/bbc/simorgh/#running-e2e-outside-eu)                         | `true`, `false`                                           |
```

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**